### PR TITLE
added support for @scope folders

### PR DESCRIPTION
**Pull request by Istenes**
Hello,
I added support for @scope packages. You can read about scopes here: https://docs.npmjs.com/misc/scope

Would appreciate if you test this PR, merge and do another npm publish.

Cheers

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,7 +14,12 @@ module.exports = function npmLinkCheck(pathToRoot, cb) {
 };
 
 function globNodeModules(startPath, cb) {
-    var pathToNodeModules = path.join(startPath, 'node_modules');
+    // if we have a scope we should check if the modules inside the folder is linked 
+    if (startPath.includes('@')) {
+        var pathToNodeModules = startPath;
+    } else {
+        var pathToNodeModules = path.join(startPath, 'node_modules');
```
Comment (etpinard) [inline index.js]: I'm not a fan of defining a `var` inside an if-else clause. 

Would you mind rewriting this as:

```js
var pathToNodeModules;

if (startPath.includes('@')) {
    pathToNodeModules = startPath;
} else {
    pathToNodeModules = path.join(startPath, 'node_modules');
}
```

or

```js
var pathToNodeModules = startPath.includes('@')
    ? startPath
    : path.join(startPath, 'node_modules');
```

?

File: test/test.js [hunk 2]
```diff
@@ -114,6 +114,35 @@ describe('nested project', function() {
...
+
+        setTimeout(done, DELAY);
+    });
+
+    it('should have 2 paths detected', function() {
+        assert.equal(pkgList.length, 2);
+    });
+
+    it('should have 2 linked paths detected', function() {
+        assert.equal(foundPathList.length, 2);
```
Comment (etpinard) [inline test/test.js]: Looks good. Thanks!

**Comment by Istenes** [comment]
Hey man,
Great! I'll fix the if else variable, missed running jslint :)

Niklas Istenes
CEO at Retroid Interactive AB

mobile: +46 731-54 48 39
twitter: @nistenes <https://twitter.com/nistenes>

web: http://retroidinteractive.com
twitter: @retroidofficial <https://twitter.com/retroidofficial>

Drottninggatan 52
SE-411 07 Göteborg
Sweden

**Comment by Istenes** [comment]
Alright, issues should be addressed now @etpinard!

**Comment by etpinard** [comment]
@Istenes thanks again!

I just published `v1.2.0` with your patch.

**Pull request merged**